content/reference/plugins/grafana/index.md

```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
 
 ### Part 2
```

Comment:
linked-accounts: we can see okta-test-ggaly

Reply:
What do you suggest?
* rename it in the dev instance?
* rename it in the html?
* remove this item?

Reply:
@ccedric

Reply:
I think we can rename it to "My Enterprise Private Login"